language/move-prover/tests/sources/functional/global_vars.no_opaque_exp

```diff
@@ -0,0 +1,103 @@
+Move prover returns: exiting with boogie verification errors
```

Comment:
We can make all the calls `public` to get rid this separated baseline file for `no_opaque`.

But on a second thought, this file might be useful as well when cross-comparing it with the baseline version. Although I am not sure what exactly this buys us other than the spec vars are corrected updated.

Reply:
I keep this; the difference is by design, and also I do not expect the no_opaque test suite to stay around for long.